apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/DeviceSelection/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { useIsFocused, useNavigation } from "@react-navigation/native";
+import { Trans } from "react-i18next";
+import { Device } from "@ledgerhq/live-common/hw/actions/types";
+import { BluetoothRequired } from "@ledgerhq/errors";
+import { Flex, Text } from "@ledgerhq/native-ui";
+import { ScreenName } from "~/const";
+import SelectDevice2, { SetHeaderOptionsRequest } from "~/components/SelectDevice2";
+import TrackScreen from "~/analytics/TrackScreen";
+import DeviceActionModal from "~/components/DeviceActionModal";
+import {
+  BaseComposite,
+  ReactNavigationHeaderOptions,
+  StackNavigatorProps,
+} from "~/components/RootNavigator/types/helpers";
+
+import { useAppDeviceAction } from "~/hooks/deviceActions";
+import { AppResult } from "@ledgerhq/live-common/hw/actions/app";
+import { WalletSyncNavigatorStackParamList } from "~/components/RootNavigator/types/WalletSyncNavigator";
+
+type NavigationProps = BaseComposite<
+  StackNavigatorProps<WalletSyncNavigatorStackParamList, ScreenName.WalletSyncActivationProcess>
+>;
+
+type Props = NavigationProps;
+
+// Defines here some of the header options for this screen to be able to reset back to them.
+export const headerOptions: ReactNavigationHeaderOptions = {
+  headerShown: false,
+};
+
+type ChooseDeviceProps = Props & {
+  isFocused?: boolean;
+  goToFollowInstructions: (device: Device) => void;
+};
+
+const APP_NAME = "Ledger Sync";
```

Comment:
this should now be part of `hw-trustchain` lib

Reply:
Yep, I'll do it on a separate PR for both platforms